### PR TITLE
I made a bash script that will play a random sound file on commit.

### PR DESCRIPTION
**Pull request by rchasman**
I thought maybe you would like this simple hook I made. If not that's okay, I really just wanted to say thanks for git-hooks and git-achievements... paired with guthug they made learning git really fun.

Thanks!

~Roey

### CODE REVIEW

**Comment by Krettis** [comment]
Are these sounds license-free?

File: contrib/commit-msg/happysounds [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#            |   |(   ) |   )|   )|  |  (   )(   )|  | |  |(   | `--.
+#            '   ' `-'`-|`-' |`-' `--|   `-'  `-' `--`-'  `-`-'`-`--'
+#                       |    |       ;
+#                       '    '    `-'
+#                              Commit sounds for your happiness
+#                 By Roey Chasman
+
+function run_test
+{
+    DIR=$HOME"/git-hooks/res/"
```
Comment (Krettis) [inline contrib/commit-msg/happysounds]: local vars would be better.

Comment (jalcine) [inline contrib/commit-msg/happysounds]: Word, and perhaps `"${HOME}/git-hooks/res"` as well.

File: contrib/commit-msg/duplicate-signedoffby [hunk 2]
```diff
@@ -1,44 +0,0 @@
-#!/bin/bash
```
Comment (Krettis) [inline contrib/commit-msg/duplicate-signedoffby]: No deletion :(

File: contrib/commit-msg/happysounds (hunk 1 above)
Comment (Krettis) [inline contrib/commit-msg/happysounds]: And maybe it is better to use an external directory as resource. Use a git config, this way you can play any sound.
Check on the available sounds in the directory perhaps.

File: contrib/pre-receive/freespace [hunk 3]
```diff
@@ -1,43 +0,0 @@
-#!/bin/bash
```
Comment (Krettis) [inline contrib/pre-receive/freespace]: No Deletion :(

File: contrib/commit-msg/duplicate-signedoffby (hunk 2 above)
Comment (rchasman) [reply contrib/commit-msg/duplicate-signedoffby]: I totally forgot this was an OSS contribution! Put it back

**Pull request closed without merging**